src/traceability/azure-traceability-helper.ts

```diff
@@ -33,12 +29,10 @@ function getAksResourceContext(): AksResourceContext {
 
 function getDeploymentPayload(deploymentReport: DeploymentReport): any {
   return {
-    // "location": "westus", // Should we set it? If yes, how?
     "properties": {
       "targetResource": {
         "name": deploymentReport.targetResource.properties['name'],
```

Comment:
Let's remove `name` per this spec:
https://github.com/Azure/azure-rest-api-specs-pr/pull/2541